BenchmarkSample/benchmark/src/androidTest/AndroidManifest.xml

```diff
@@ -18,6 +18,8 @@
     xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
     package="com.example.benchmark.test">
+
+    <!-- [START benchmark_manifest] -->
```

Comment:
Note that this isn't needed anymore in benchmark 1.1, as we set test variant to release via gradle plugin.

May still want to document this in guidance docs, but only for 1.0 users, and probably doesn't need to be linked to sample - can just remove it here.

Reply:
It's different between microbench and macrobench. In microbench (this manifest), starting in 1.1:

- You aren't expected to create a benchmark buildType
- Plugin defaults you to the release buildType for tests (which sets debuggable false, and uses release library dependencies (important for avoiding coverage codegen))
- Profileable is only needed for profiling feature

Back in 1.0, I think you would need to force debuggable=false like this, and we hadn't solved the default test type = release, but I think we can just leave this out given how close we are to stable 1.1

I think it's way simpler to avoid differentiating 1.0, given how much it'll complicate things for such a short time